### ISSUE TITLE
Issue with updating 

### ISSUE THREAD

**Issue by matitoprzechvj** [koogar/Gnat-Stats#9]
Hey, when i run hardwareserialmonitor and select right com, screen shows correctly my hardware, temps, usage, etc. but then it updates and everything is broken. It looks like it is pulling more information that it needs or something. I attached video showing the problem.
Im using ESP32 DEV with SH1106 1.3" 128x64 display
My PC: ryzen 7 5700x, nvidia rtx 4080

https://github.com/koogar/Gnat-Stats/assets/122699823/bb30a74a-7da9-4a36-90ab-68ef575813d9

**Comment by koogar** [comment]
Looks like the data gets out of sync you could try altering 

//>>>>>>>>>>>>>>>>>>>>>>>>>>>
/* Delay screen event, to help stop screen data corruption ESP8622 use 25, most others 5 will do*/
int Serial_eventDelay = 15;

In the Configuration.h

if you are running a USB hub plug it directly into the PC.
Try a different USB Lead

**Comment by matitoprzechvj** [comment]
I've tried it and it didn't work. I've also tried different ports, different cables, even different esp board. I also don't think that this issue is related to my PC because, I've tried it on another PC and the behavior was the same. That C210 and MC405 shown on video are cpu clock and gpu memory clock, infos that i don't need. For me, I want only the basic info like its displaying in first seconds before it updates. Any more ideas? I really want this project to work, it looks awesome

**Comment by matitoprzechvj** [comment]
Okay, fixed it. I flashed the BT version and it works even with serial. Turns out that Base version is (I think) trying to switch between display styles but in the folder there is only one style and that's creating the issue. When i flashed the v1.7 bt version with 4 styles it works perfectly (almost). My cpu usage and gpu usage seems off. For example in task manager my cpu usage is 20% but on the screen it displays only 1-2%, same with gpu.
